### ISSUE TITLE
GroupTags.txt can't have more than 9 options

### ISSUE THREAD

**Issue by nkh1105** [matbe/PowerShell#4]
Hi Matt,

I was able to narrow down to what the issue is.  Everything works great up until you add a 10th item to GroupTags.txt.  Once you get 10 or more items in the GroupTags.txt file, that is when you start seeing the weird behavior. The first Tag works and any tag beyond 10 works, but nothing in between works, you get the error "Not a valid Selection" In my previous post I had uploaded 13 different Tags into the GroupTags.txt file. I hope that was enough info, please let me know if you need anything else.  I really do love this script, thank you for creating it.

Thanks,
Nick

**Comment by matbe** [comment]
Hi Nick,

Thank you for reporting but I cannot replicate this issue, I just tried with 30 items and it worked for me. Could you please share a GroupTags.txt file that has the issue? Here or on a site like pastebin maybe?

What Windows-version are you trying to install 1909?

**Comment by nkh1105** [comment]
Hi Matt,
I have this in pastebin, link is below.  I appreciate your help and support.    

https://pastebin.com/embed_js/zMHzPU0t

Thanks,
Nick

**Comment by matbe** [comment]
Thank you, I was able to replicate the issue now. I do not fully understand why it happens but I created a workaround that hopefully solves the problem. It should be fixed in commit 8856b5a .

I also have built a function that allows for multipage selection if you have a large number of groupstags (more that can fit the screen). And another solution that dynamically get grouptags based on AzureAD groups by using a prefix on the group name and then uses the descriptionfield for the tag. Let me know if any of those are of interest and I'll try and make the userfriendly :)

Let me know if the workaround fixes your problem and I will close this issue.
